Podcast/SearchViewController.swift

```diff
@@ -72,6 +72,12 @@ class SearchViewController: ViewController, UISearchControllerDelegate, UITableV
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
         searchController?.searchBar.isHidden = false
+        searchResultsController.subviewsWillAppear()
```

Comment:
why are we calling this?

Reply:
This forces the tabs to call updateTableViewInsetsForAccessoryView()